ddl/src/main/java/com/adaptivescale/rosetta/ddl/targets/kinetica/KineticaDDLGenerator.java

```diff
@@ -85,6 +85,10 @@ public String createTable(Table table, boolean dropTableIfExists) {
         }
 
         String tableType = extractTableType(table);
+        String partitions = table.getPropertyAsString("partitions");
+        if (partitions != null) {
```

Comment:
suggestion: using hash maps containsKey method should be safer alternative

Reply:
I have modified those parts based on your suggestion.